fastlane/lib/fastlane/actions/sh.rb

```diff
@@ -1,8 +1,16 @@
+
+require_relative '../helper/sh_helper.rb'
 module Fastlane
   module Actions
     class ShAction < Action
       def self.run(params)
         # this is implemented in the sh_helper.rb
+        Actions.sh_control_output(
```

Comment:
This is one thing I had to do to get sh() to work for some reason. I can't figure out how it worked without it.

Reply:
Oh I see, it's in `fastfile.rb`